docker-compose-prod.yml

```diff
@@ -20,18 +20,18 @@ services:
       - database
 
   database:
-    image: mysql:5.7
-    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+    image: mysql:8-oracle
```

Comment:
I took this image as per CAE adoption to it. If prefer something else, please suggest.